### PR TITLE
Nahiyan_Dark mode on all modal in User Management

### PR DESCRIPTION
**Pull request by Nahiyan-16**
# Description
All modals are now fully integrated with dark mode on the user management page

## Related PRS (if any):
This frontend PR is related to the dev backend

## Main changes explained:
- All modals have dark mode

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard→ Other Links -> User Management
6. verify all modals are working properly. 

## Screenshots or videos of changes:
Screenshot 2024-05-08 130011
Screenshot 2024-05-08 130034

### CODE REVIEW

**Review by Ramaruva — CHANGES_REQUESTED** [review]
Hi @Nahiyan-16  I have checked all modal in user management page all modals are in dark mode except for the change password modal I am not sure whether the change password have darkmode or not?
image


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/eaca57f3-f149-4d96-88e5-0031fa0874fa

(https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/b92c80de-c84e-47ff-8fdd-d8496c6b2ce2)

and also I have one suggestion that can we make this icon "X" has white in dark mode instead of black because as a user I feel I am unable to see properly  "X" icon in dark mode

**Review by MinSun0814 — DISMISSED** [review]
The updated dark mode looks fantastic and aligns with the aesthetic of the application. Nice work!
image
image
image

**Review by linkhanthtel — DISMISSED** [review]
I reviewed your PR and all modals are working properly. Nice work! 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/95618937/e314b610-75b3-4ae1-93e7-6bcb2df94751

**Comment by Nahiyan-16** [comment]
> Hi @Nahiyan-16 I have checked all modal in user management page all modals are in dark mode except for the change password modal I am not sure whether the change password have darkmode or not? image
> 
>  usermodel.mp4 
> (https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/b92c80de-c84e-47ff-8fdd-d8496c6b2ce2)
> 
> and also I have one suggestion that can we make this icon "X" has white in dark mode instead of black because as a user I feel I am unable to see properly "X" icon in dark mode

Fixed the issue regarding dark mode on password modal and added new custom header with the X mark being able to toggle from light to dark

**Comment by Ramaruva** [comment]
> > Hi @Nahiyan-16 I have checked all modal in user management page all modals are in dark mode except for the change password modal I am not sure whether the change password have darkmode or not? image
> > usermodel.mp4
> > (https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/b92c80de-c84e-47ff-8fdd-d8496c6b2ce2)
> > and also I have one suggestion that can we make this icon "X" has white in dark mode instead of black because as a user I feel I am unable to see properly "X" icon in dark mode
> 
> Fixed the issue regarding dark mode on password modal and added new custom header with the X mark being able to toggle from light to dark

Hi I verified the latest changes now I am able to see dark mode for reset password modal great work on the X  mark also
image
image
image

**Review by Ramaruva — DISMISSED** [review]
latest changes are working

**Comment by ZhangPeizhou** [comment]
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/95693786/6575370a-00c2-4c08-9a93-783035d1f292

Everything works as expected, nice work!

**Comment by ZhangPeizhou** [comment]
I want to mention that when I click to view a user profile, it automatically switches to light mode, and I have to switch it back manually. I am not sure if it needs to be fixed here or not, just let you know.

**Review by nathcarnevalli — DISMISSED** [review]
The Dark mode worked as expected in User Management

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/69649453/01cebc5a-6119-4b66-8781-2fb0dc7f555a

**Review by hbodgal — DISMISSED** [review]
image

it works well.

File: src/components/UserManagement/logTimeOffPopUp.jsx [hunk 1]
```diff
@@ -4,7 +4,6 @@ import moment from 'moment-timezone';
 import {
   Button,
   Modal,
-  ModalHeader,
```
Comment (abdel-lall) [inline src/components/UserManagement/logTimeOffPopUp.jsx:7]: This error was caused by the deletion of this import.
Screenshot 2024-05-11 230729

**Review by abdel-lall — CHANGES_REQUESTED** [review]
The Edit time-off request modal do not change color in dark mode.

Screenshot 2024-05-11 230639

**Review by cgomezhub — APPROVED** [review]
well done!
All the popups works in dark mode.

https://www.loom.com/share/e8e1cd66176143be8c7f11b8f7a63e19?sid=467ac1c1-95ef-4bdb-afb6-f5f7c9a34f57

**Review by sucheta90 — COMMENTED** [review]
Hello @Nahiyan-16,

After reviewing this PR, I see the feature works as expected. Please refer to the video. However, I noticed that refreshing the page resets the dark mode back to light mode. Is this the intended behavior?

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/101291283/75114daa-5e92-4c02-9300-98ab39f53659

**Comment by Nahiyan-16** [comment]
> Hello @Nahiyan-16,
> 
> After reviewing this PR, I see the feature works as expected. Please refer to the video. However, I noticed that refreshing the page resets the dark mode back to light mode. Is this the intended behavior?
> 
>  PR2238_review.mov

No this is not the intended behavior and will be dealt with in another PR.

**Review by peter6866 — APPROVED** [review]
All modals in user management work great in dark modal. 
image
image
image
image

**Comment by one-community** [comment]
Thank you all, merging!

**Pull request merged**